dotCMS/src/main/java/com/dotcms/auth/providers/saml/v1/DotSamlResource.java

```diff
@@ -0,0 +1,232 @@
+package com.dotcms.auth.providers.saml.v1;
+
+import com.dotcms.rest.annotation.NoCache;
+import com.dotcms.saml.Attributes;
+import com.dotcms.saml.DotSamlConstants;
+import com.dotcms.saml.DotSamlProxyFactory;
+import com.dotcms.saml.IdentityProviderConfiguration;
+import com.dotcms.saml.IdentityProviderConfigurationFactory;
+import com.dotcms.saml.SamlAuthenticationService;
+import com.dotcms.saml.SamlException;
+import com.dotmarketing.exception.DoesNotExistException;
+import com.dotmarketing.util.Logger;
+import com.dotmarketing.util.WebKeys;
+import com.liferay.portal.model.User;
+import org.glassfish.jersey.server.JSONP;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.io.IOException;
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+
+@Path("/v1/dotsaml")
+public class DotSamlResource implements Serializable {
```

Comment:
Is there any place where these endpoints are documented? For example, it is not clear for me the difference between `doLogin` and `processLogin`